eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/mutable/UnmodifiableBag.java

```diff
@@ -463,4 +467,56 @@ protected Object readResolve()
     {
         return this.getMutableBag().selectUnique();
     }
+
+    @Override
+    public <V> MutableObjectLongMap<V> sumByInt(Function<? super T, ? extends V> groupBy, IntFunction<? super T> function)
```

Comment:
Is this a new implementation on `UnmodifiableBag`? If it is then why are we adding it in the same PR? If it is not a new implementation, then why is it needed?

Reply:
I think this is because the specialization of these methods requires concrete implementations and `UnmodifiableBag` does not extend the `AbstractMutableBagIterable` class. Please confirm @prathasirisha. Thanks!

Reply:
That is right @donraab.

Reply:
@prathasirisha can you check in the test coverage if this is covered? If not there should be a new test to cover this.

Reply:
@prathasirisha can you please check if this and other new implementations are covered by test coverage. If not we should have new unit tests to cover it.

Reply:
Hi @nikhilnanivadekar, thank you for reviewing this code again. I was able to confirm that these new methods are covered by running the tests with coverage in my IDE.  

That said, I am wondering if we should turn back the coverage action build for individual forks. I see a benefit of it in cases like this, where we could simply download the coverage report (from our personal forks) to ensure the new methods are covered before merging them to upstream repo. Let me know your thoughts on this. Happy to make the changes if you too think it is a good idea.